src/components/VesselDashboardGoogleMap.vue

```diff
@@ -157,13 +154,21 @@ export default {
 
   watch: {
     newSelectedReport (newValue) {
-      if(!this.fetchingReports) {
-        this.googleMapMarkers.forEach( googleMapMarker => {
+      if(!this.fetchingReports && this.googleMapMarkers.length > 0) {
+        this.googleMapMarkers.forEach(googleMapMarker => {
           googleMapMarker.setIcon(POINT_MARKER_ICON_CONFIG)
+          googleMapMarker.setAnimation(null)
         })
+
         const selectedMarker = this.googleMapMarkers.find(marker => marker.marker.id === newValue.id)
         selectedMarker.setIcon(SELECTED_POINT_MARKER_ICON_CONFIG)
       }
+
+      this.dashboardGoogleMap.panTo({ lat: newValue.lat, lng: newValue.lng })
```

Comment:
What do you think about destructuring `newValue` in function params and use it here?